LearningMissionSimulation/PlaygroundGavril.cs

```diff
@@ -206,7 +213,11 @@ public void CreateClassrooms(int classroomCount)
                 {
                     Guid moduleId = moduleIdList[AttributeGenerator.random.Next(0, moduleIdList.Count)];
                     Module module;
-                    moduleDictionary.TryGetValue(moduleId, out module);
+                    var moduleFound = moduleDictionary.TryGetValue(moduleId, out module);
+                    if (! moduleFound  )
```

Comment:
Combine these two lines in one:
Replace:
 var moduleFound = moduleDictionary.TryGetValue(moduleId, out module);
                    if (! moduleFound  )
with:
          if(!moduleDictionary.TryGetValue(moduleId, out module))

Reply:
Done